gollum.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'gollum-lib', '~> 5.0.a'
   s.add_dependency 'kramdown', '~> 1.9.0'
   s.add_dependency 'sinatra', '~> 2.0'
+  s.add_dependency 'sinatra-contrib', '~> 2.0'
```

Comment:
Not sure how relevant this is, but from the README of sinatra-contrib: 

NOTE: This project has been merged upstream to sinatra/sinatra.

To use this gem from the upstream repository, you can add the following to your Gemfile:
```ruby
github 'sinatra/sinatra' do
  gem 'sinatra-contrib'
end
```

Reply:
Cool! One less dependency